openwisp_users/api/filters.py

```diff
@@ -0,0 +1,159 @@
+import swapper
```

Comment:
Let's not call this file `filters`, because that would be confused with other type of classes which are called filter and are a different thing (https://www.django-rest-framework.org/api-guide/filtering/#generic-filtering).

I suggest naming this file `mixins.py`, remember to update the docs.